### PR TITLE
[AWS][Lambda] Fix AWS Invocations Count Issue

### PR DESCRIPTION
**Pull request by agithomas**
- Bug

## Proposed commit message

Update aggregation function for AWS lambda invocation metric.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] Dashboard validation
- [x] Discover view validation
- [x] Field mapping validation
- [x] Package upgrade test 

## How to test this PR locally

- [x] elastic-package build
- [x] elastic-package stack up -v -d --services package-registry

## Screenshots

image

image

### CODE REVIEW

**Review by dej611 — APPROVED** [review]
Code review only 👍

**Comment by niraj-elastic** [comment]
Let's update the screen shot as well.

File: packages/aws/kibana/dashboard/aws-7ac8e1d0-28d2-11ea-ba6c-49a884eb104f.json [hunk 1]
```diff
@@ -371,24 +362,23 @@
                                                     "sourceField": "aws.dimensions.FunctionName"
                                                 },
                                                 "dd394e42-4f85-4c30-891d-ad1b0e127a8e": {
+                                                    "customLabel": false,
                                                     "dataType": "number",
                                                     "isBucketed": false,
-                                                    "label": "Maximum of aws.lambda.metrics.Invocations.avg",
-                                                    "operationType": "max",
+                                                    "label": "Sum of aws.lambda.metrics.Invocations.sum",
+                                                    "operationType": "sum",
```
[the remark this answers is not in the document]
Comment (agithomas) [reply packages/aws/kibana/dashboard/aws-7ac8e1d0-28d2-11ea-ba6c-49a884eb104f.json:369]: @niraj-elastic , the field is of metric-type `gauge`. Not a counter. 

Hence, your statement `now in next fetch there are no new invocations so in second document value of Invocations.sum : 10` is not true. 

Below is the AWS Webconsole screenshot using the `Invocation.sum` metric value. 

image

**Comment by agithomas** [comment]
> Let's update the screen shot as well.

Updated the screenshot.

File: packages/aws/kibana/dashboard/aws-7ac8e1d0-28d2-11ea-ba6c-49a884eb104f.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (niraj-elastic) [reply packages/aws/kibana/dashboard/aws-7ac8e1d0-28d2-11ea-ba6c-49a884eb104f.json:369]: Makes sense.

**Review by muthu-mps — APPROVED** [review]

**Pull request merged**